tests/model_serving/model_server/utils.py

```diff
@@ -223,3 +231,136 @@ def run_inference_multiple_times(
 
             if exceptions:
                 raise InferenceResponseError(f"Failed to run inference. Error: {exceptions}")
+
+
+def verify_keda_scaledobject(
+    client: DynamicClient,
+    isvc: InferenceService,
+    expected_trigger_type: str | None = None,
+    expected_query: str | None = None,
+    expected_threshold: str | None = None,
+) -> None:
+    """
+    Verify the KEDA ScaledObject.
+
+    Args:
+        client: DynamicClient instance
+        isvc: InferenceService instance
+        expected_trigger_type: Expected trigger type
+        expected_query: Expected query string
+        expected_threshold: Expected threshold as string (e.g. "50.000000")
+    """
+    scaled_objects = get_isvc_keda_scaledobject(client=client, isvc=isvc)
+    scaled_object = scaled_objects[0]  # Get the first ScaledObject
+    trigger_meta = scaled_object.spec.triggers[0].metadata
+    trigger_type = scaled_object.spec.triggers[0].type
+    query = trigger_meta.get("query")
+    threshold = trigger_meta.get("threshold")
+
+    assert trigger_type == expected_trigger_type, (
+        f"Trigger type {trigger_type} does not match expected {expected_trigger_type}"
+    )
+    assert query == expected_query, f"Query {query} does not match expected {expected_query}"
+    assert threshold == expected_threshold, f"Threshold {threshold} does not match expected {expected_threshold}"
+
+
+def run_vllm_concurrent_load(
+    isvc: InferenceService,
+    num_concurrent: int = 5,
+    duration: int = 120,
+) -> None:
+    """
+    Run VLLM concurrent load.
```

Comment:
This comment only says what's already in the function name. Try to write docstrings that explain a bit more, e.g.
```suggestion
    Send several inference requests concurrently to a vLLM inference service
```